src/js/disability-benefits/components/ClaimsListItem.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+export default function ClaimsListItem({ claim }) {
+  const filesNeeded = claim.attributes.trackedItems.stillNeedFromYouList ? claim.attributes.trackedItems.stillNeedFromYouList.length : null;
+  return (
+    <Link className="claim-list-item" to={`/your-claims/${claim.attributes.evssId}/status`}>
```

Comment:
Let's use our internal ID here, the route that you will call to get the details will use that ID too